client/src/components/TODOList/TODOCalendarIcon.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { ExclamationCircleIcon } from "@heroicons/react/20/solid";
+import {
+  Popover,
+  PopoverContent,
+  PopoverHandler,
+} from "@material-tailwind/react";
+import TODOItem from "./TODOItem";
+
+// Creates an icon within the current "Date" tile to show the presence of
+// TODOItems starting at that date. Clicking the icon will show a popover
+// list of all the TODOItems for that "Date" tile
+const TODOCalendarIcon = ({ TODOList }) => {
+  return (
+    <Popover>
+      <PopoverHandler>
+        <ExclamationCircleIcon className="fill-red-500" />
```

Comment:
Perhaps finished TODOs should not trigger a red exclamation mark since it feels as though such a symbol should indicate that something still needs to be done.

Reply:
The red icon is just to let the user know that there are todo items associated with that particular date tile. I think that is fine. I can perhaps change the colour if that is what you want.

Reply:
Perhaps if a day has unfinished TODOs have it be red but if it only has finished TODOs change it to some other color?

Reply:
I like that idea. But this might need to be done after the milestone 5 demo. Would that be okay?